packages/storybook-test/stories/link/link.stories.tsx

```diff
@@ -146,3 +230,129 @@ export const ExtremelyLongLink: Story = {
     status: { type: [] },
   },
 };
+
+export const Hover: Story = {
+  name: 'Link:hover',
+  args: {
+    children: 'voorbeeldsite (opent in een nieuw venster)',
+    className: 'nl-link--hover',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met :hover state.',
+      },
+    },
+    status: { type: [] },
+  },
+};
+
+export const Active: Story = {
+  name: 'Link:active',
+  args: {
+    children: 'voorbeeldsite (opent in een nieuw venster)',
+    className: 'nl-link--active',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met :active state.',
+      },
+    },
+    status: { type: [] },
+  },
+};
+
+export const FocusVisible: Story = {
+  name: 'Link:focus-visible',
+  args: {
+    children: 'voorbeeldsite (opent in een nieuw venster)',
+    className: 'nl-link--focus-visible',
+    href: 'https://example.com',
+    target: '_blank',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een link met :focus-visible state.',
+      },
+    },
+    status: { type: [] },
+  },
+};
+
+export const Current: Story = {
+  name: 'Link (current page)',
+  args: {
+    children: 'voorbeeldsite (opent in een nieuw venster)',
+    className: 'nl-link--current',
+    href: 'https://example.com',
+    target: '_blank',
```

Comment:
Zou dit niet de React APi `current="page"` of `current` (boolean) moeten gebruiken, want `aria-current` moet ook ingesteld worden.